library.properties

```diff
@@ -1,5 +1,5 @@
 name=Low-Power
-version=1.6
+version=1.81
```

Comment:
Should it be `1.8`?

Reply:
https://github.com/rocketscream/Low-Power/blob/master/README.md says 1.81

Reply:
Oh right, the release download has a 1.80 in the `README.md`, but the current master version says `1.81`.  There is one fix commit between 1.80 and 1.81.